docs/integrations/engines/snowflake.md

```diff
@@ -549,3 +549,13 @@ MODEL (
   ),
 );
 ```
+
+## Troubleshooting
+
+### Frequent Authentication Prompts
+
+When using Snowflake with enabled security features like Multi-Factor Authentication (MFA), you may experience repeated prompts for authentication tokens or sign-ins while running SQLMesh commands. This typically occurs when your Snowflake account is configured to restrict short-lived token issuance. To reduce authentication prompts, you can enable token caching in your Snowflake connection configuration:
```

Comment:
When using Snowflake with security features like Multi-Factor Authentication (MFA), you may experience repeated prompts for authentication while running SQLMesh commands. This typically occurs when your Snowflake account is configured to issue short-lived tokens. 

To reduce authentication prompts, you can enable token caching in your Snowflake connection configuration: